docs/DeveloperGuide.md

```diff
@@ -854,6 +854,46 @@ Prerequisite: Only 1 guest to be edited. The guest's index should exist.
    3. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `edit ...`, `edit 1 n/Jo@`<br>
       Expected: No guest is edited, because the **parameters are invalid**. Error details shown in the result display.
 
+   5. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `edit ...`<br>
+      Expected: No guest is edited, because the parameters are invalid. Error details shown in the status message.
+      Status bar remains the same.
+
+### Marking Guests Rooms as Unclean
+
+Prerequisite: All guests room cleaning status will be edited.
+
+   1. Test case: `markroomsunclean`<br>
+      Expected: All guests are marked as unclean, the icon representing `isRoomClean` will change to red from green.
```

Comment:
Perhaps "Expected: All guests' room clean statuses are marked as unclean and the icon representing `isRoomClean` will change to red for all the guests."